pincer/objects/events/guild.py

```diff
@@ -133,9 +132,9 @@ class GuildMemberRemoveEvent(APIObject, GuildProperty):
     user: User
 
     def __post_init__(self):
-        self.user = User.from_dict(
-            construct_client_dict(self._client, self.user)
-        )
+        # NOTE: Idk if this works, but I got a type Error otherwise
+        # TODO: Test this
+        self.user = User.from_dict({**self.user})
```

Comment:
concern

Reply:
idk how to test it

Reply:
dirty trick, but `User.from_dict(vars(self.user))` should work